security/hcaptcha.go

```diff
@@ -14,7 +14,11 @@ import (
 	"time"
 )
 
-type CaptchaResponse struct {
+type GotrueRequest struct {
+	Security GotrueSecurity `json:"__gotrue_meta_security"`
```

Comment:
hmm i'm not too sure if it's a good practice for api design to prefix a request body param with "__" ? was wondering it's supposed to signify?

Reply:
I don't know it to be bad. It signifies "private" or "internal" a la Python; here it indicates that this field is gotrue-internals specific, vs the other fields in the requests which are likely going to be reflected in the user db or upstream provider APIs.

Feel free to change it, though.